Makefile.core.mk

```diff
@@ -354,6 +354,7 @@ refresh-goldens:
 	@REFRESH_GOLDEN=true go test ${GOBUILDFLAGS} ./pkg/kube/inject/...
 	@REFRESH_GOLDEN=true go test ${GOBUILDFLAGS} ./pilot/pkg/security/authz/builder/...
 	@REFRESH_GOLDEN=true go test ${GOBUILDFLAGS} ./cni/pkg/plugin/...
+	@REFRESH_GOLDEN=true go test ${GOBUILDFLAGS} ./istioctl/cmd/...
```

Comment:
this is probably accidental?

Reply:
Dunno. This was added a few versions ago. I don't even know what this does (if anything). You think I should remove it?

Reply:
nevermind, I thought it might be a rebase artifact, but I vaguely recall something about this being needed. let's leave it in